Issue 3 Remove/Hide Edit button for DONE ideas

ikakus opened the pull request:
Hide "Edit" button if task is set to done.

**Fixes**: #3

ikakus committed:
Added comment for method getTabById()

ikakus committed:
Added comment for method getTabById()

ikakus committed:
Hiding "Edit" button if task is set to done

ikakus committed:
Hiding "Edit" button if task is set to done

ikakus committed:
Added flags when calling MainActivity intent fixing issue #1

Commit:
Changed size of Heading to 24dp

Commit:
Added Spacing

ikakus committed:
Changing color of "Plus" buttons text when changing color of text fixing issue #4

ikakus committed:
Not hiding edit button, but just disabling the edit.

ikakus committed:
Not hiding edit button, but just disabling the edit.

naushad-madakiya commented on app/src/main/java/manparvesh/ideatrackerplus/recycler/RecyclerOnClickListener.java:
```diff
@@ -73,35 +73,39 @@ private void showIdeaDialog() {
-
+        final int state = mDbHelper.getTabById(mIdRecycler);
+
+        LovelyStandardDialog lovelyStandardDialog =
```
Why create object when it's not being used anywhere else? It's better to use `.show`  on the dialog.

naushad-madakiya commented on app/src/main/java/manparvesh/ideatrackerplus/recycler/RecyclerOnClickListener.java:
```diff
@@ -73,35 +73,39 @@ private void showIdeaDialog() {
-                .setNegativeButtonColorRes(R.color.md_pink_a200)
-                .show();
-
+        final int state = mDbHelper.getTabById(mIdRecycler);
```
It can be if we move `state` inside the edit callback, as it's not necessary that state will be in use every time.

naushad-madakiya reviewed (changes requested):
Added minor changes to improve performance, Also if we don't change the order unnecessary that can help better to track changes. i.e. changing the order of negative and neutral callbacks.

naushad-madakiya committed:
Merge pull request #56 from ikakus/issue_4_text_color_changing

Change color of fab when text color changes issue: #4

Commit:
Merge branch 'issue_1_back_takes_back_to_intro' of https://github.com/ikakus/IdeaTrackerPlus into ikakus-issue_1_back_takes_back_to_intro

Commit:
Merge branch 'ikakus-issue_1_back_takes_back_to_intro'

Commit:
Create CODE_OF_CONDUCT.md

Commit:
Create LICENSE

manparvesh committed:
Merge pull request #59 from vinitkumar/patch-1

Create CODE_OF_CONDUCT.md and LICENSE files

Commit:
chore(gitignore): add proper .gitignore and remove .idea files

The `.gitignore` was incomplete, so added a proper .gitignore from
here. https://github.com/github/gitignore/blob/master/Android.gitignore

Also, `.idea` files should never be pushed to git as it will cause
conflicts and useless commits that need not be there as everyone will
have a slighly different once.
- Github Issue: Fixes #60

Signed-off-by: Vinit Kumar <vinit.kumar@socialschools.nl>

Commit:
update .gitignore

mukund54 committed:
Fixed edit button in idea (#43)

- Display 'save' as a text instead of 'edit' in Idea Edit dialog.
Fix #14

naushad-madakiya committed:
Merge pull request #46 from KrishAmal/issue_10_bigger_heading

Changed size of Heading

Commit:
fix conflicts

naushad-madakiya committed:
Merge pull request #61 from vinitkumar/fix/gitignore

chore(gitignore): add proper .gitignore and remove .idea files

ikakus committed:
Not adding "Edit" button to dialog if idea status is "Done"

ikakus committed:
Merge remote-tracking branch 'origin/issue_3_editing_done_task' into issue_3_editing_done_task

# Conflicts:
#	app/src/main/java/manparvesh/ideatrackerplus/recycler/RecyclerOnClickListener.java

naushad-madakiya commented:
@ikakus can you check the changes requested?

ikakus commented:
@naushad-madakiya Sure. I think it would be better if we merge this one with #58 because it changes the whole dialog, so we'd have to disable "Edit" in new dialog too.
